### ISSUE TITLE
2025-11-21: sgl-project/sglang 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#443]
# 每日更新报告（2025-11-21）

## sgl-project/sglang

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-11-21 23:01:23 | Mick | [diffusion] log: minor improve logging (#13735) |
| 2025-11-21 22:26:39 | fzyzcjy | Super tiny delete unused files (#13734) |
| 2025-11-21 21:42:32 | Lianmin Zheng | Fix url: use https://roadmap.sglang.io for roadmap (#13733) |
| 2025-11-21 19:02:36 | weibingo | fix 3fs down, lock schedule main thread (#13407) |
| 2025-11-21 19:00:27 | Mick | [diffusion] doc: add contributing.md (#13649) |
| 2025-11-21 18:48:28 | jacky.cheng | [AMD] Enable fused shared expert append and flatten quant for fp8 deepseekR1 model (#13705) |
| 2025-11-21 18:47:16 | Mick | [diffusion] profile: support performance metric dumping and comparison (#13630) |
| 2025-11-21 18:15:46 | Xiaoyu Zhang | [Clean code] Compressed_tensors_moe code clean (#13719) |
| 2025-11-21 17:11:37 | Cheng Wan | [11/N] MoE Refactor: Simplifying SBO Implementation with Dispatcher Hooks (#13327) |
| 2025-11-21 17:07:08 | Even Zhou | [NPU] chore: bump to CANN 8.3.RC1 and Pytorch 2.8.0 (#13647) |
| 2025-11-21 16:01:33 | Ziming Huang | [Fix] Qwen3Next lmhead dtype  (#13708) |
| 2025-11-21 15:40:31 | Teng Ma | [HiCache] fix unit test with changed new APIs (#13498) |
| 2025-11-21 15:32:40 | Michele Marzollo | [Bugfix] Fix hidden state size in EAGLE PD disaggregation buffers (#13590) |
| 2025-11-21 15:17:15 | Lianmin Zheng | [Auto Sync] Update http_server.py, io_struct.py, scheduler_... (20251120) (#13679) |
| 2025-11-21 15:14:24 | zyksir | [diffusion] server: use meta to avoid Linear init for TextEncoder (#13564) |
| 2025-11-21 14:54:41 | ishandhanani | Fix ZMQ bind error on non-zero rank nodes when using SGLANG_BLOCK_NONZERO_RANK_CHILDREN=0 (#13686) |
| 2025-11-21 14:35:29 | Mick | [diffusion] doc: minor update docs (#13177) |
| 2025-11-21 14:26:33 | Yuhao Yang | [difusion] CI: speed up multimodal_gen ci (#13665) |
| 2025-11-21 13:28:38 | Stefan He | [Piecewise Cuda Graph] rename, refactor and add more logging (#13675) |
| 2025-11-21 13:21:51 | Yuan Luo | [VLM] Support Piecewise CUDA Graph for InternVL (#13640) |
| 2025-11-21 13:16:42 | Xiaoyu Zhang | [opt kimi k2 4 / n] Delete useless pad kernel in sgl_moe_align_block_size (#13587) |
| 2025-11-21 13:15:27 | Xiaoyu Zhang | [kimi k2 thinking] Avoid useless torch.zeros_  (#13596) |
| 2025-11-21 12:16:05 | Mike Qiu | Fix target MLA with eagle3 support for PD disaggregation (#13555) |
| 2025-11-21 11:10:51 | Glen Liu | remove unnecessary starvation check (#13619) |
| 2025-11-21 11:02:42 | alisonshao | Add sgl-kernel CI test for Blackwell (B200) (#13301) |
| 2025-11-21 10:30:12 | Kaixi Hou | [NVIDIA] Add cutedsl e2e test to GB200 CI (#12672) |
| 2025-11-21 10:00:02 | alisonshao | [CI] Move nightly tests to test/nightly/ (#13683) |
| 2025-11-21 09:57:17 | Chang Su | [model-gateway] fix gateway cli arg parser to not use = (#13685) |
| 2025-11-21 09:13:13 | Mick | [diffusion] CI: improve validation method (#13627) |
| 2025-11-21 09:00:28 | Simo Lin | [model-gateway] add both python and rust cli alias (#13678) |
| 2025-11-21 07:50:16 | 赵晨阳 | Update quantization.md with new model resources (#13677) |
| 2025-11-21 07:35:12 | fzyzcjy | Super tiny remove unused MiniMaxM2MLP class (#13659) |
| 2025-11-21 06:20:11 | Stefan He | [Piecewise CUDA Graph] Fix recompile issue for Mixtral and Grok2 (#13667) |
| 2025-11-21 05:41:58 | alisonshao | Test reorganization: Move tests to manual/ (#13610) |
| 2025-11-21 05:34:32 | Glen Liu | allow loras to be implicitly evicted and loaded based on max_loaded_loras (#11526) |
| 2025-11-21 04:55:13 | alisonshao | Add nightly test CI monitor workflow (#13038) |
| 2025-11-21 04:53:02 | b8zhong | enable csgmv automatically on cuda (#13600) |
| 2025-11-21 04:24:10 | hlu1 | [Deepseek V3.2] Change indexer weights_proj to fp32 (#13459) |
| 2025-11-21 04:04:30 | Vincent Zhong | Fix launch of `Olmo3` (#13666) |
| 2025-11-21 03:13:36 | YAMY | [DeepseekV3.2] Deepseek fp8 support for MHA path (#12964) |
| 2025-11-21 02:04:36 | Douglas Yang | Adding nightly tests as release guard for bot bump workflows (#13655) |
| 2025-11-21 01:45:51 | Fan Yin | [10/n] decouple quantization impl from vllm dependency - fix import (#13524) |
| 2025-11-21 01:33:31 | YanbingJiang | Add fused_rmsnorm_gated_cpu kernel for CPU to support Qwen3-Next (#11577) |
